Gemfile

```diff
@@ -11,3 +11,7 @@ end
 if File.exists?(File.join(Dir.home, '.gemfile'))
   instance_eval(File.read(File.join(Dir.home, '.gemfile')))
 end
+
+group :development do
+  gem 'vmpooler', git: 'https://github.com/puppetlabs/vmpooler.git', branch: 'vmpooler-dns-gcp'
+end
```

Comment:
Replace this before taking out of draft mode please. Also, add a new line at the end of the file.